frontend/src/Frontend/UI/DeploymentSettings.hs

```diff
@@ -1002,12 +1028,22 @@ uiDeployPreview model settings keys accounts signers gasLimit ttl code lastPubli
   where
     uiPreviewResponses signing (cmd, wrappedCmd) = do
       pb <- getPostBuild
+
+      unless (getAny $ foldMap (Any . any isGas) capabilities) $ do
```

Comment:
Why not `any (any isGas) capabilities` ?

Reply:
Why not indeed!